Sources/CoreCommands/SwiftTool.swift

```diff
@@ -664,51 +664,67 @@ public final class SwiftTool {
         return buildSystem
     }
 
+    private func _buildParams(toolchain: UserToolchain) throws -> BuildParameters {
+        let targetTriple = toolchain.targetTriple
+
+        // Use "apple" as the subdirectory because in theory Xcode build system
+        // can be used to build for any Apple platform and it has it's own
```

Comment:
nit
```suggestion
        // can be used to build for any Apple platform and it has its own
```

Reply:
I think the comment should probably be moved to the implementation here, where it does seem appropriate:

```
public func platformBuildPathComponent(buildSystem: BuildSystemProvider.Kind) -> String {
        buildSystem == .xcode ? "apple" : self.platformBuildPathComponent()
    }
```